template_feed/Microsoft.DotNet.Common.ItemTemplates/content/ViewImports/.template.config/template.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "ASP.NET"
+  ],
+  "name": "MVC ViewImports",
+  "generatorVersions": "[1.0.0.0-*)",
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.ViewImports",
+  "precedence": "100",
+  "identity": "Microsoft.AspNetCore.Mvc.ViewImports",
+  "shortName": "viewimports",
+  "sourceName": "_ViewImports",
+  "primaryOutputs": [
+    {
+      "path": "_ViewImports.cshtml"
+    }
+  ],
+  "defaultName": "_ViewImports",
```

Comment:
/cc @seancpeters - right now this is always outputing the file `content.cshtml`. What did I do wrong here?